packages/components/tsconfig.json

```diff
@@ -11,7 +11,7 @@
     "outDir": "dist",
     "paths": {
       "@jobber/hooks/*": [
-        "../../packages/hooks/src/*"
```

Comment:
If we look in the src directory it means we have to parse the typescript twice, which was causing a lot of issues in Rollup. Looking at the dist folder means we look at built files instead. I believe this was the root of why we needed to do our old jobber/hooks juggling in order to get it loading.